src/views/Rules/Rule/index.jsx

```diff
@@ -412,7 +543,7 @@ export default function Rule({ isNewRule, ...props }) {
           </Grid>
         </Fragment>
       )}
-      {!isLoading && (
+      {!isLoading && !actionLoading && (
```

Comment:
Ah, my bad. This should actually be:

```diff
diff --git a/src/views/Rules/Rule/index.jsx b/src/views/Rules/Rule/index.jsx
index 992d119..042f084 100644
--- a/src/views/Rules/Rule/index.jsx
+++ b/src/views/Rules/Rule/index.jsx
@@ -94,12 +94,12 @@ export default function Rule({ isNewRule, ...props }) {
   const [updateSCAction, updateSC] = useAction(updateScheduledChange);
   const [deleteSCAction, deleteSC] = useAction(deleteScheduledChange);
   const isLoading =
-    fetchRuleAction.loading ||
-    scheduledChangeAction.loading ||
-    products.loading ||
-    channels.loading;
+    fetchRuleAction.loading || products.loading || channels.loading;
   const actionLoading =
-    addSCAction.loading || updateSCAction.loading || deleteSCAction.loading;
+    addSCAction.loading ||
+    updateSCAction.loading ||
+    deleteSCAction.loading ||
+    scheduledChangeAction.loading;
   const error =
     fetchRuleAction.error ||
     scheduledChangeAction.error ||
@@ -543,10 +543,11 @@ export default function Rule({ isNewRule, ...props }) {
           </Grid>
         </Fragment>
       )}
-      {!isLoading && !actionLoading && (
+      {!isLoading && (
         <Fragment>
           <Tooltip title={isNewRule ? 'Create Rule' : 'Update Rule'}>
             <Fab
+              disabled={actionLoading}
               onClick={isNewRule ? handleCreateRule : handleUpdateRule}
               color="primary"
               className={classNames({
@@ -559,7 +560,7 @@ export default function Rule({ isNewRule, ...props }) {
           {hasScheduledChange && (
             <SpeedDial ariaLabel="Secondary Actions">
               <SpeedDialAction
-                disabled={scheduledChangeAction.loading}
+                disabled={actionLoading}
                 icon={<DeleteIcon />}
                 tooltipOpen
                 tooltipTitle="Cancel Pending Change"
```

Reply:
Ah, to disable the controls instead of hide them? I was wondering about that :)

Reply:
Yeah :)